src/managers/GuildMemberRoleManager.js

```diff
@@ -81,6 +81,17 @@ class GuildMemberRoleManager {
     return this.cache.find(role => role.premiumSubscriberRole) || null;
   }
 
+  /**
+   * The managed role this member created when joining the guild, if any
+   * <info>Only ever available on bots</info>
+   * @type {?Role}
+   * @readonly
+   */
+  get botRole() {
+    if (!this.member.user.bot) return null;
```

Comment:
```suggestion
    if (!this.client.user.bot) return null;
```
incase for whatever reason the bots GuildMember object is uncached

Reply:
No, the purpose of this is to target the member the GuildMemberRoleManager belongs to and is [not nullable](https://discord.js.org/#/docs/main/stable/class/GuildMemberRoleManager?scrollTo=member), this would check if the client user is not a bot, which is obsolete and not the point of the check.

Reply:
oh right, misread that, mb